web/crux-ui/locales/en/registries.json

```diff
@@ -32,7 +32,11 @@
     "gitlab": "Add your GitLab Registry.",
     "github": "Add your GitHub Registry.",
     "google": "Add your Google Registry.",
-    "unchecked": "Add a registry without authentication. This type does not provide image nor tag browsing."
+    "unchecked": "Add a registry without authentication. This type does not provide image nor tag browsing.",
+    "learnMorePat": "Learn more about personal access token",
+    "learnMoreRegistry": "Learn more about the registry",
+    "githubTokenReason": "Github allow downloads, while access to the complete listing of all tags, which we extensively utilize, requires authentication.",
```

Comment:
For me `Github allows downloads, while access to the complete listing of all tags requires authentication, which we extensively utilize.` sounds more "natural". But maybe we could ask @gerimate to write a GitHub token explanation?

Reply:
Github allows downloads, while access to the complete listing of all tags requires authentication, which we extensively utilize. --- add after the Learn more section.

Reply:
![image](https://github.com/dyrector-io/dyrectorio/assets/9247788/6d4b5001-1419-49e8-8972-3b2a408ad5cb)

Changed to this one.